config/ape.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | coordinape_user_address
+    |--------------------------------------------------------------------------
+    |
+    | This value is the address of the Coordinape User. This user is added to
+    | every circle by default and receives GIVE during epochs.
+    |
+    */
+
+    'coordinape_user_address' => env('COORDINAPE_USER_ADDRESS'),
```

Comment:
could we replace `DEV_ETH_ADDRESS` in env.example with this

Reply:
done